frontend/src/bundles/common/components/loader/loader.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Circle, Flex, Text } from '@chakra-ui/react';
+
+import { SPIN_ANIMATION } from './libs/constants/constants.js';
+
+const Loader = (): JSX.Element => {
+    return (
+        <Flex flexDirection="column" alignItems="center">
+            <Box position="relative" width="100px" height="100px">
+                <Circle
+                    size="full"
+                    backgroundColor="white"
+                    color="text.default"
+                >
+                    LOGO
+                </Circle>
+                <Circle
+                    position="absolute"
+                    inset="0"
+                    borderWidth="5px"
+                    borderColor="shadow.200"
+                    borderTopColor="brand.secondary.300"
+                    animation={`${SPIN_ANIMATION} 1s linear infinite`}
+                ></Circle>
+            </Box>
+            <Text fontSize="lg" marginTop="10px">
```

Comment:
Maybe we should use the shortcuts that chakra ui is providing us with (e.g. alignItems -> align, marginTop -> mt)?

Reply:
I don't like shortcasts because they are hard to read